EpsilonLights/Src/Lights.c

```diff
@@ -20,15 +20,15 @@ void updateLightsTask(void const* arg)
     {
         osDelayUntil(&prevWakeTime, LIGHTS_UPDATE_FREQ);
         headlightsOff = (lightsInputs >> HOFF_INPUT_INDEX) & 1;
-        headlightsLow = (lightsInputs >> HLOW_INPUT_INDEX) & 1;
-        headlightsHigh = (lightsInputs >> HHIGH_INPUT_INDEX) & 1;
+        headlightsLow = (lightsInputs >> HLOW_INPUT_INDEX) & 1; //0b0000x0010
+        headlightsHigh = (lightsInputs >> HHIGH_INPUT_INDEX) & 1; //0b0000x0100
```

Comment:
Remove these comments

Reply:
Orr they can be expanded on to be useful :)